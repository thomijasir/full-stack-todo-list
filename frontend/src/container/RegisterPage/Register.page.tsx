import React from 'react';

const RegisterPage = () => {
  return (
    <div>
      <h1>Its Register Screen</h1>
    </div>
  );
};

export default RegisterPage;
