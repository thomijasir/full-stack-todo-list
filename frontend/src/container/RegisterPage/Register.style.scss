.register-page {
  display: flex;
  align-items: center;
  padding-top: 40px;
  padding-bottom: 40px;
  background-color: #f5f5f5;
  height: inherit;
  .form-register {
    width: 100%;
    max-width: 330px;
    padding: 15px;
    margin: auto;
  }

  .form-register {
    .form-floating:focus-within {
      z-index: 2;
    }
    .form-username,
    .form-email,
    .form-password {
      margin-bottom: -1px;
      border-bottom-right-radius: 0;
      border-bottom-left-radius: 0;
    }
    .form-re-password {
      margin-bottom: 10px;
      border-top-left-radius: 0;
      border-top-right-radius: 0;
    }
  }
}
