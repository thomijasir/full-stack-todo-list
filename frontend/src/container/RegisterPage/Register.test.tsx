import React from 'react';
import renderer from 'react-test-renderer';
import RegisterPage from './Register.page';

test('Render Register Page', () => {
  const element = renderer.create(<RegisterPage />).toJSON();
  expect(element).toMatchSnapshot();
});
