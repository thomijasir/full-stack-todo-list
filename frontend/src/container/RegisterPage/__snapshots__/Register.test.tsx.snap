// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Register Page 1`] = `
<div>
  <h1>
    Its Register Screen
  </h1>
</div>
`;
